src/components/controls/link/link-editor.vue

```diff
@@ -0,0 +1,98 @@
+<template> 
+    <div id="linkEditorModal" v-if="modalOpen" class="modal fade show overflow-auto" tabindex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" style="padding-right: 17px; display: block;">
```

Comment:
can you use better name instead of example?

Reply:
please also add two classes `scbd-controls link-editor` these classes should added to all controls at root level